src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/HotReload/IProjectHotReloadSession.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.HotReload
+{
+    public interface IProjectHotReloadSession
+    {
+        bool IsSupported { get; }
```

Comment:
@BillHiebert Is this needed? If Hot Reload isn't supported for some reason, I would just as soon have `IProjectHotReloadAgent.CreateHotReloadSession` return `null`.

Reply:
@tmeschter we could remove this and do as you say.